### ISSUE TITLE
always requre LB service for every pack

### ISSUE THREAD

**Issue by nzhang1220** [oneops/setup#31]
just tried the latest OneOps all-in-one vagrant installation, and it seems that no matter which pack I use, the deployment will require `lb` service, otherwise the deployment will not happen. For example, even for `Java`, `lb` service is required (In fact, `lb` should not be needed at all).

**Comment by ighouri** [comment]
@nzhang1220 .. load balancer service should have required for redundant environment in OneOps .. So are you trying to deploy redundant environment in OneOps (please check environment configuration in Transition) ? If so, I assume you could simply just add lb service in your cloud. Right ?

**Comment by nzhang1220** [comment]
@ighouri I think I was using Single. Even for redundant, if the pack does not explicitly require LB like below, we should not need to add LB service, right?

:requires => { "constraint" => "1..1", "services" => "lb,dns" },

**Comment by ighouri** [comment]
@nzhang1220 Yes if either the environment is configured as 'single' or if the pack does not require lb explicitly, then we do not need to have lb service. 
Please clarify your use case that you are trying and and will see why it is asking for lb service for you.

**Comment by nzhang1220** [comment]
To reproduce this issue:

install latest OneOps Vagrant all-in-one-box.
Create a java design in single env.
Without adding LB service, the deployment will report "LB service is missing"

That is how I did.

Thanks
